src/analysis/signals.rs

```diff
@@ -89,7 +89,6 @@ fn analyze_signal(
 
     let action_emit_name = if signal.is_action {
         imports.add("glib");
```

Comment:
Not needed any longer (maybe grep for that generally here). Also `use gio` would not be needed if we have that somewhere

Reply:
It's actually the last one remaining. :)